packages/arb-token-bridge-ui/src/generateOpenGraphImages.tsx

```diff
@@ -0,0 +1,345 @@
+      width={width}
+      height={height}
+      alt="Arbitrum Logo"
+    />
+  )
+}
+
+function getCoreChainImage(from: Chain, to: Chain) {
+  return (
+    <div
+      style={{
+        ...dimensions,
+        display: 'flex',
+        position: 'relative',
+        width: dimensions.width,
+        height: dimensions.height,
+        backgroundColor: 'black',
+        fontFamily: 'Unica77'
+      }}
+    >
+      <div
+        style={{
+          transform: 'rotate(180deg)',
+          mixBlendMode: 'overlay',
+          backgroundImage:
+            'url("https://arbitrum.io/background/grains_bottom.png")',
```

Comment:
Can we have it as part of this repo, we had issues in the past where images were moved on arbitrum.io

Reply:
relative url does not work with satori.... only possible alternatives are
1. change the element layout, and use this png in base64 form, or
2. keep the image here, and do not use the current version but refer to prod `bridge.arbitrum.io` version

I think given that it's just grains for artistic effect, it's ok to use arbitrum.io, it's also not so bad if it's gone missing because the bg would just become pure black

Reply:
Can we move it to bridge.arbitrum.io?